webook/utils/collision_analysis.py

```diff
@@ -0,0 +1,88 @@
+from collections import namedtuple
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Dict, List, Tuple, Union
+
+import pytz
+from django.db.models import Q
+
+from webook.arrangement.models import Arrangement, Event, EventSerie, Room
+
+
+@dataclass
+class CollisionRecord:
+    event_a_title: str
+    event_a_start: datetime
+    event_a_end: datetime
+    event_b_title: str
+    event_b_start: datetime
+    event_b_end: datetime
+    contested_resource_id: int
+    contested_resource_name: str
+
+
+Range = namedtuple('Range', ['start', 'end'])
```

Comment:
What is this used for?

Reply:
I have removed Range in development. It was an oversight on my part and should have been removed. Used it to store Start-End ranges previously.